src/unk_0202055C.c

```diff
@@ -0,0 +1,51 @@
+#include "heap.h"
+#include "unk_0202055C.h"
+
+static Unk0202055C *sUnknown;
+
+void sub_0202055C(u32 a0, HeapID a1) {
```

Comment:
a1 can be named `heap_id` (or `heapId` depending on your preference)

Reply:
Done